### ISSUE TITLE
meta_data function

### ISSUE THREAD

**Issue by ailnlv** [joaoqalves/web2py-oauth2#8]
Hi. In `controllers/token.py` and `controllers/protected_resource.py` you reference a function called `meta_data`. However, this function isn't defined anywhere. 
Shouldn't this function be defined in 'utils.py'? What is it supposed to do exactly?

**Comment by joaoqalves** [comment]
@ailnlv Yes. There was a function that did that. This code was merged with a fork that @SamuelMarks did. The function was present in the [first version](https://github.com/joaoqalves/web2py-oauth2/blob/e780b6ebb54bdcf4cbb578e8060680ad49bfa66f/models/utils.py).
